.devcontainer/setup.sh

```
@@ -0,0 +1,105 @@
+            "module": "pytest",
+            "args": ["-v", "-s"],
+            "console": "integratedTerminal",
+            "justMyCode": false
+        }
+    ]
+}
+EOF
+fi
+
+echo "📦 Instalando Crawl4AI en modo desarrollo..."
+cd /workspace
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)